File: rust/test2/src/fileTest.rs
use std::fs;

const FILE_PATH = "D:/newlang/rust/test.txt";

pub fn test(){
	let a= fs::read_to_string(FILE_PATH).unwrap();
	println!("{}", a);
}

pub fn test2(){
	let a = fs::read(FILE_PATH).unwrap();
	for k in a{
		println!("{}", k);
	}
}

pub fn test3(){
	fs::write(FILE_PATH, "apple\nbanana\norange").unwrap();
}

File: python/basic/fileTest.py

def format(line):
	s = line.strip()
	s = s.split(" ")
	print(str("'%s'" %s[1]), end=", ")

def test():
	a=open("a.txt", encoding="utf-8")
	for line in a:
		print(str('"%s"' %line.strip()), end=", ")
	a.close()

def test2():
	a=open("data/test.txt", mode='w')
	for k in ("Apple", "Banana", "Orange"):
		a.write(k + "\n")
	a.close()

def test3():
	a=open("data/test.txt", mode='a')
	for k in ("Pear", "Guava", "Grape"):
		a.write(k + "\n")
	a.close()

def test4():
	a=open("data/test.txt")
	print(a.read())
	a.close()

def test5():
	with open("a.txt", encoding="utf-8") as a:
		for line in a:
			format(line)

test5()

